pkg/apis/aws/v1alpha1/shared_types.go

```diff
@@ -152,3 +152,9 @@ var DefaultConfigMap = "aws-account-operator-configmap"
 
 // DefaultConfigMapAccountLimit holds the fallback limit of aws-accounts
 var DefaultConfigMapAccountLimit = 100
+
+// AWSManagedRoleName is the name of the read-only role
+var AWSManagedRoleName = "AWSManagedRole"
+
+// AWSManagedRoleArn is the rean-only ARN for the AWSManagedRole
+var AWSManagedRoleArn = "arn:aws:iam::aws:policy/ReadOnlyAccess"
```

Comment:
I think this might work well as a first pass, but as CEE's needs evolve and as we add more roles I'm not convinced this approach will scale.

Reply:
I completely agree with this. The reason I did it this way was because I was trying to remain within the scope of the card.